python_modules/libraries/dagster-dg/dagster_dg/cli/scaffold.py

```diff
@@ -97,10 +97,13 @@ def __init__(self, *args, **kwargs):
     def get_command(self, ctx: click.Context, cmd_name: str) -> Optional[click.Command]:
         if not self._commands_defined and cmd_name not in HARDCODED_COMMANDS:
             self._define_commands(ctx)
+
+        # First try exact match
         cmd = super().get_command(ctx, cmd_name)
-        if cmd is None:
-            exit_with_error(generate_missing_plugin_object_error_message(cmd_name))
-        return cmd
+        if cmd is not None:
+            return cmd
```

Comment:
per discussion, also match on exact class name match ignoring package/module in a follow up